### ISSUE TITLE
Switch GPU command missing server address error

### ISSUE THREAD

**Issue by jathanasiou** [quebin31/nvman#2]
Hi there. I just installed nvman and gave it a go. After enabling optimus-manager, I've been stuck at the status shown below. My system runs Antergos with Cinnamon and an Nvidia 1050 Mobile.

```
└─▪ nvman status
Optimus: active (enabled)
Bumblebee: inactive (disabled)
Default service: optimus
Default startup for optimus is 
:
Currently using 
: with optimus
```

While attempting to use `sudo nvman switch intel|nvidia` gives the following error:

```
└─▪ sudo nvman switch nvidia

Switching to mode : nvidia
Logging out the current desktop session
Traceback (most recent call last):
  File "/usr/bin/optimus-manager", line 11, in <module>
    load_entry_point('optimus-manager==1.0', 'console_scripts', 'optimus-manager')()
  File "/usr/lib/python3.7/site-packages/optimus_manager/optimus_manager_client.py", line 78, in main
    gpu_switch(config, switch_mode)
  File "/usr/lib/python3.7/site-packages/optimus_manager/optimus_manager_client.py", line 113, in gpu_switch
    sessions.logout_current_desktop_session()
  File "/usr/lib/python3.7/site-packages/optimus_manager/sessions.py", line 12, in logout_current_desktop_session
    session_bus = dbus.SessionBus()
  File "/usr/lib/python3.7/site-packages/dbus/_dbus.py", line 211, in __new__
    mainloop=mainloop)
  File "/usr/lib/python3.7/site-packages/dbus/_dbus.py", line 100, in __new__
    bus = BusConnection.__new__(subclass, bus_type, mainloop=mainloop)
  File "/usr/lib/python3.7/site-packages/dbus/bus.py", line 122, in __new__
    bus = cls._new_for_bus(address_or_type, mainloop=mainloop)
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.BadAddress: Server address of type unix was missing argument path or abstract
```

I tried doing the above switch without sudo, and in this case my session does logout+login without error (at least none I can spot before I'm logged out), but the `nvman status` remains the same.

Any idea why that might be?

**Comment by quebin31** [comment]
Does `optimus-manager` works for you? Because this looks like a problem with `optimus-manager`, remember `nvman` is just a wrapper, open an issue on [optimus-manager repository](https://github.com/Askannz/optimus-manager).

**Comment by jathanasiou** [comment]
The optimus-manager switch commands seem to work fine for them most part, even though nvidia-to-intel can be a bit iffy. But so does its startup setting command.  

It's only when doing them via nvman that I get this error.

**Comment by quebin31** [comment]
I can't reproduce your error, what's your desktop specification and enviroment?

**Comment by jathanasiou** [comment]
Hey, I got some updates pushed to Antergos today and this seems seem to be fixed with `nvman: 1.2.2-1`. I think this can be closed.

**Comment by quebin31** [comment]
Yeah, I forgot to close this issue, I fixed it in version `1.2.2`
